package.json

```diff
@@ -15,7 +21,11 @@
     "vite": "^4.1.4"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/free-solid-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "sass": "^1.58.3"
```

Comment:
`sass` should not be a devDependency I think.

Reply:
I sent a voice.

Reply:
Done.